server/webapp/WEB-INF/rails/webpack/helpers/spark_routes.ts

```diff
@@ -241,4 +241,11 @@ export default class {
   static apiAdminAccessTokenRevokePath(id: number) {
     return `${this.apiAdminAccessTokensBasePath()}/${id}/revoke`;
   }
+
+  static apiAdminAccessClusterProfilesPath(id?: string) {
```

Comment:
any reason for a single method for `GET` and `INDEX`?

Reply:
No specific reason. Few have separate methods for `GET` and `INDEX` few have a single method. Which approach should we follow?